src/content/i18n/pt-BR.yml

```
@@ -31,7 +31,7 @@ recipesLink.plural: Receitas relacionadas
 # 404 Page
 404.title: Não encontrado
 404.content: Esta página não está no nosso sistema solar.
-404.linkText: Me leve para casa.
+404.linkText: Página inicial.
```

The original is "Take me home", I guess we can still keep the original context but not translate "home" directly. Like this:
```suggestion
404.linkText: Me leve para página inicial.
```

---

That makes sense